publications.md

```diff
@@ -136,7 +136,7 @@ Previous works have shown that differentiable physics simulators provide gradien
     
     <tr>
         <td class="pubimg" width="250" height="100">
-            <img src="../pics/pouring.gif" img width="250" height="100">
+            <img src="../pics/pouring.gif" img width="250" height="125">
```

Comment:
Does this actually change the height? The td parent tag has height=100. I think 250 x 100 is standard for the gifs, not sure if this should be changed.